From: Wim Van Sebroeck <wim@iguana.be>
Subject: watchdog: watchdog_core.h: make functions extern
References: 
Patch-Mainline: v3.5-rc1
Git-commit: fb5f6658163412dce22724e906e324ab7fd62c18


Signed-off-by: Thomas Renninger <trenn@suse.de>

Make the functions in watchdog_core.h extern like it should.

Signed-off-by: Wim Van Sebroeck <wim@iguana.be>

diff --git a/drivers/watchdog/watchdog_core.h b/drivers/watchdog/watchdog_core.h
index bc7612b..80503f2 100644
--- a/drivers/watchdog/watchdog_core.h
+++ b/drivers/watchdog/watchdog_core.h
@@ -29,5 +29,5 @@
 /*
  *	Functions/procedures to be called by the core
  */
-int watchdog_dev_register(struct watchdog_device *);
-int watchdog_dev_unregister(struct watchdog_device *);
+extern int watchdog_dev_register(struct watchdog_device *);
+extern int watchdog_dev_unregister(struct watchdog_device *);
